### PR TITLE
Removing access_token from query param

### PR DESCRIPTION
**Pull request by roflc0pter**
**Changes:**
- the rest call from the client (Yourturn Frontend) sends the token as authentication header instead of query parameter
- the route `/tokeninfo` and the middleware `oauth` send the `access_token` as authorization header instead of query parameter

### CODE REVIEW

**Comment by roflc0pter** [comment]
:+1:

**Comment by neremic** [comment]
:+1:

**Comment by roflc0pter** [comment]
:+1:

**Comment by roflc0pter** [comment]
:+1:

**Comment by neremic** [comment]
:+1:

**Comment by roflc0pter** [comment]
:+1:

File: server/src/routes/tokeninfo.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ var winston = require("winston"),
 
 function info(req, res) {
   var accessToken = req.get("authorization");
+  console.log("accessToken", accessToken);
```
Comment (hjacobs) [inline server/src/routes/tokeninfo.js]: this should not be here

**Comment by roflc0pter** [comment]
**Changes:**
- the rest call from the client (Yourturn Frontend) sends the token as authentication header instead of query parameter
- the route `/tokeninfo` and the middleware `oauth` send the `access_token` as authorization header instead of query parameter

File: server/src/routes/tokeninfo.js (hunk 1 above)
Comment (roflc0pter) [reply server/src/routes/tokeninfo.js]: That was while wip for debugging.

**Comment by roflc0pter** [comment]
:+1:

**Comment by zeddmalam** [comment]
👍

**Comment by roflc0pter** [comment]
:+1:

**Comment by piknyc** [comment]
👍

**Comment by piknyc** [comment]
👍

**Pull request merged**